src/main/java/core/basesyntax/db/FruitMap.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitMap {

```

Comment:
`FruitMap` sounds like your custom banana-driven implementation of Map. Please, rename.